hack/multikueue-e2e-test.sh

```diff
@@ -65,20 +64,18 @@ function startup {
 }
 
 function kind_load {
+    prepare_docker_images
     if [ $CREATE_KIND_CLUSTER == 'true' ]
     then
-        docker pull $E2E_TEST_IMAGE
         cluster_kind_load $MANAGER_KIND_CLUSTER_NAME
         cluster_kind_load $WORKER1_KIND_CLUSTER_NAME
         cluster_kind_load $WORKER2_KIND_CLUSTER_NAME 
-
+    fi
```

Comment:
👍 Seems right based on ref where image is loaded on kind before installing jobset ([ref](https://github.com/kubernetes-sigs/kueue/blob/cea42f23253a4a3221ed529faef539cb61ec742f/hack/multikueue-e2e-test.sh#L74-L84))